components/yotpo/sources/new-reviews/new-reviews.js

```diff
@@ -0,0 +1,34 @@
+const yotpo = require('../../yotpo.app.js')
+module.exports = {
+  name: 'yotpo-new-reviews',
+  version: '0.0.1',
+  props: {
+    http: {
+      type: "$.interface.http",
+    },
+    sendTestEvent: {
+      type: "boolean",
+      label: "Send Test Event",
+      description: "Create a test review when this source is activated",
+      optional: true,
+      default: true,
+    },
+    yotpo,
+  },
+  hooks: {
+    async activate() {
+      await this.yotpo.createWebhook("review_create", this.http.endpoint)
+      await this.yotpo.createWebhook("review_updated", this.http.endpoint)
+      if (this.sendTestEvent) {
+        await this.yotpo.sendTestEvents()
+      }
+    },
+    async deactivate() {
+      await this.yotpo.deleteWebhook("review_create", this.http.endpoint)
+      await this.yotpo.deleteWebhook("review_updated", this.http.endpoint)
+    },
+  },
+  async run(event) {
+    this.$emit(event)
```

Comment:
I'd recommend adding a `summary` to the emit metadata. Are there also conditions where Yotpo might send the same event twice (on retry or other cases)? Normally I'll add a `dedupe` strategy at the top of the component, then pass an `id` in the emit metadata that the deduper uses (see Component API docs).